data/fields/house.json

```diff
@@ -1,5 +1,16 @@
 {
     "key": "house",
     "type": "combo",
-    "label": "House Type"
+    "label": "House Type",
+    "strings": {
+        "options": {
+            "bungalow": "Bungalow",
+            "detached": "Detached House",
+            "link-detached": "Link-Detached House",
```

Comment:
`house=link-detached` has only ~750 uses and less than 20 in the `building` key. There isn't a preset for it in iD. It is documented though, so it might be a good idea to create a preset.